server/src/main/java/com/cloud/server/StatsCollector.java

```diff
@@ -219,6 +224,8 @@ public String toString() {
     private static final ConfigKey<String> statsOutputUri = new ConfigKey<String>("Advanced", String.class, "stats.output.uri", "",
             "URI to send StatsCollector statistics to. The collector is defined on the URI scheme. Example: graphite://graphite-hostaddress:port or influxdb://influxdb-hostaddress/dbname. Note that the port is optional, if not added the default port for the respective collector (graphite or influxdb) will be used. Additionally, the database name '/dbname' is  also optional; default db name is 'cloudstack'. You must create and configure the database if using influxdb.",
             true);
+    private static final ConfigKey<Double> secondaryStorageCapacityThreshold = new ConfigKey<Double>("Advanced", Double.class, "secondary.storage.capacity.threshold", "0.90",
+            "Secondary storage capacity threshold (1 = 100%).", true);
```

Comment:
@GutoVeronezi can you move this config to 'CapacityManager' ? and use it wherever applicable.

also, update StorageOrchestrator class with this config, it is hardcoded here: https://github.com/apache/cloudstack/blob/a0d9acec6f3926ef31430a2be0c2737b30eb5ebb/engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/StorageOrchestrator.java#L104

Reply:
Done.